x-pack/test/functional/apps/ml/pages.ts

```diff
@@ -53,5 +53,17 @@ export default function ({ getService }: FtrProviderContext) {
       await ml.dataVisualizer.assertDataVisualizerImportDataCardExists();
       await ml.dataVisualizer.assertDataVisualizerIndexDataCardExists();
     });
+
+    it('it should load the stack management with the ML menu item being present', async () => {
+      await ml.navigation.navigateToStackManagement();
+    });
+
+    it('it should load the jobs list page in stack management', async () => {
+      await ml.navigation.navigateToStackManagementJobsListPage();
+    });
+
+    it('it should load the analytics jobs list page in stack management', async () => {
```

Comment:
nit: "it" at the beginning of the sentences seems redundant 🙂

Reply:
Of course 😅 Fixed in 0a1325e.